docs/data/datamodel.md

```diff
@@ -1,8 +1,333 @@
+| `alternativeTitles` | lang_string[]    | 0-n         |                                                         |
+| `urls`              | url[]            | 0-n         |                                                         |
+| `additional`        | lang_string\|url | 0-n         |                                                         |
+
+
+#### Person
+
+| Field            | Type     | Cardinality | Restrictions                           |
+| ---------------- | -------- | ----------- | -------------------------------------- |
+| `__id`           | string   | 1           |                                        |
+| `__type`         | string   | 1           | Literal 'Person'                       |
+| `givenNames`     | string[] | 1-n         |                                        |
+| `familyNames`    | string[] | 1-n         |                                        |
+| `jobTitles`      | string[] | 0-n         |                                        |
+| `affiliations`   | id[]     | 0-n         | Organization IDs                       |
+| `address`        | address  | 0-1         |                                        |
+| `email`          | string   | 0-1         |                                        |
+| `secondaryEmail` | string   | 0-1         |                                        |
+| `authorityRefs`  | url[]    | 0-n         | References to external authority files |
+
+
+#### Organization
+
+| Field             | Type        | Cardinality | Restrictions                           |
+| ----------------- | ----------- | ----------- | -------------------------------------- |
+| `__id`            | string      | 1           |                                        |
+| `__type`          | string      | 1           | Literal 'Organization'                 |
+| `name`            | string      | 1           |                                        |
+| `url`             | url         | 1           |                                        |
+| `address`         | address     | 0-1         |                                        |
+| `email`           | string      | 0-1         |                                        |
+| `alternativeName` | lang_string | 0-1         |                                        |
+| `authorityRefs`   | url[]       | 0-n         | References to external authority files |
+
+
+#### Grant
+
+| Field     | Type   | Cardinality | Restrictions               |
+| --------- | ------ | ----------- | -------------------------- |
+| `__id`    | string | 1           |                            |
+| `__type`  | string | 1           | Literal 'Grant'            |
+| `funders` | id[]   | 1-n         | Person or Organization IDs |
+| `number`  | string | 0-1         |                            |
+| `name`    | string | 0-1         |                            |
+| `url`     | url    | 0-1         |                            |
+
+
+### Value Types
+
+#### lang_string
```

Comment:
```suggestion
#### Language Tag (`lang_string')
```

Reply:
Thanks. I'm going for `#### String with Language Tag (lang_string)` instead though, to make it clear that it's not _only_ the tag, but the string with the tag.